### PR TITLE
Pr for slider and header feat

### PR DESCRIPTION
**Pull request by narharikale**
added header and theme
added slider

### CODE REVIEW

**Review by rahulmistry751 — APPROVED** [review]
Good folder structure and meta tags used.

File: src/frontend/context/Theme-Context/Theme-Context.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
+import { createContext, useState , useLayoutEffect , useContext} from "react";
+
+
+const themeContext = createContext(null);
+
+
+
+
```
Comment (shraddha15-creator) [inline src/frontend/context/Theme-Context/Theme-Context.js:9]: remove empty lines

File: src/frontend/context/Theme-Context/Theme-Context.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+function ThemeProvider( {children} ) {
+    const [theme , setTheme] = useState('dark');
+    
+
+
```
Comment (shraddha15-creator) [inline src/frontend/context/Theme-Context/Theme-Context.js:14]: remove empty lines

**Review by shraddha15-creator — APPROVED** [review]
Everything looks so nice and clean.
Code is well structured, clean, and easy to understand.
UI also look clean. Good Theme colors.

Got to know about ` useLayoutEffect ` hook and ` react-responsive-carousel ` and how you use it in your code.

**Review by Tejs1 — APPROVED** [review]
Nice UI
-add [package-lock.json] to gitIgnore.

File: src/frontend/context/Theme-Context/Theme-Context.js (hunk 2 above)
Comment (narharikale) [reply src/frontend/context/Theme-Context/Theme-Context.js:14]: done

**Pull request merged**